modules/planning/README.md

```diff
@@ -16,15 +16,19 @@ use cases as they were all treated as a single driving scenario.
 Apollo 5.5, which focuses on curb-to-curb autonomous driving on urban roads,
 introduced 2 new planning scenarios.
 
-Apollo 6.0, which focuses on intellegence of autonomous driving, introduced two new planning modes: E2E mode and Hybrid mode. 
+Apollo 6.0, which focuses on intellegence and data-driven development of autonomous driving, introduced two new planning modes: E2E mode and Hybrid mode, by which we demonstrate the new capability of dynamically nudginng a moving obstacle. These two modes also define a series of APIs where developers can generate thier own training data and brought in their own models.
+
+**Note** that the current development of E2E mode is in early stage where it is fined tuned more for the dynamic nudge scenario rather than comprehensive ones. So suboptimality is expected when running other scenarios in simulation. E2E mode and Hybrid mode are not tested on real vehicle yet. The work on data-driven planning is in still in progress and welcomes any collaboration with developer by any means including algorithms, models and data.
```

Comment:
1) **Note**: The current development ...

2) `fine-tuned`.  
3) `is in still in progress` => `is still in progress` .
4) `with developer` should be removed.